.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: Main
```

Comment:
Ah, I didn't realize earlier that this repo already has tests set up in GitHub actions (the `.github/workflows/tests.yml` file—that's why there are two sets of ["checks" on this PR](https://github.com/panorama-ed/scan_left/pull/67/checks)).

Your version seems slightly cleaner (though I do like the name "tests" rather than "main," but don't feel strongly). Could you make sure we only have one?

Reply:
Doh! That totally makes sense, haha. For some reason I thought that there had to be a `main.yml` and `tests.yml`.